src/mixins/prop-fill.js

```diff
@@ -1,12 +1,13 @@
 // When declaring a prop with no value it should be evaluated to true
 // but '' is falsy. As a solution add the
 export default {
-  beforeCompile () {
-    if (!this._props) return
-    for (let prop of Object.keys(this._props)) {
-      let data = this._props[prop]
-      if (data.options.fill && data.raw === '') {
-        this[prop] = prop
+  created () {
```

Comment:
I prefer you keep this the way it was since is no longer used. This way I can test things easier and see if we keep it or not